inventories/group_vars/control-plane/metal.yml

```diff
@@ -18,10 +18,10 @@ metal_api_images:
   url: https://images.metal-stack.io/metal-os/master/firewall/2.0-ubuntu/20201126/img.tar.lz4
   features:
     - firewall
-- id: ubuntu-20.04.20201126
-  name: Ubuntu 20.04 20201126
-  description: Ubuntu 20.04 20201126
-  url: https://images.metal-stack.io/metal-os/master/ubuntu/20.04/20201126/img.tar.lz4
```

Comment:
The old image needs to be kept, just add the cloud-init variant as a separate entry.